.storybook/main.ts

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+
+const language = process.env.LANGUAGE;
+
+const config: { webpackFinal: (config: any) => Promise<any>; stories: string[]; framework: { name: string }; addons: (string | { name: string; options: { sass: { implementation: any } } })[] } = {
+    stories: ['../src/**/*.stories.@(js|jsx|ts|tsx)'],
+    addons: [
+        '@storybook/addon-links',
+        '@storybook/addon-essentials',
+        '@storybook/addon-interactions',
+        {
+            name: '@storybook/addon-styling',
+            options: {
+                sass: {
+                    implementation: require('sass'),
+                },
+            },
+        },
+        '@storybook/addon-styling-webpack'
+    ],
+    framework: {
+        name: '@storybook/react-webpack5',
+    },
+    webpackFinal: async (config: any) => {
```

Comment:
do we need anything else from `webpack.config.js` like `watchOptions`?
https://github.com/box/box-ui-elements/blob/1b6f4c2885f24caa8945383a3031f628c2371486/.storybook/webpack.config.js#L11-L13

Reply:
not that im aware of, i didnt run into any other issues with webpack